### PR TITLE
ODC-6495 Remove layout2 topology

### PR DESCRIPTION
**Pull request by abhinandan13jan**
# Addresses
https://issues.redhat.com/browse/ODC-6495

# Screenshot
Screenshot 2022-03-07 at 7 20 43 PM

# Tests
no change

# Browser conformance
Chrome

### CODE REVIEW

**Comment by vikram-raj** [comment]
This Gerkin script also needs to be removed. 
https://github.com/openshift/console/blob/master/frontend/packages/topology/integration-tests/features/topology/topology-layout-save.feature#L29-L43

**Review by vikram-raj — COMMENTED** [review]
These steps also need to be removed as we do not have layout 2 button
https://github.com/openshift/console/pull/11145/files#diff-716ca63f3b7495366ae6d33e5599b12793f480a73669b6a4194c41a010d8df75L50

https://github.com/openshift/console/pull/11145/files#diff-716ca63f3b7495366ae6d33e5599b12793f480a73669b6a4194c41a010d8df75L63

File: frontend/packages/topology/src/components/graph-view/layouts/layoutFactory.ts [hunk 1]
```diff
@@ -2,17 +2,14 @@ import { Graph, Layout, LayoutFactory } from '@patternfly/react-topology';
 import TopologyColaLayout from './TopologyColaLayout';
 
 const COLA_LAYOUT = 'Cola';
-const COLA_FORCE_LAYOUT = 'ColaForce';
 
 const layoutFactory: LayoutFactory = (type: string, graph: Graph): Layout | undefined => {
   switch (type) {
-    case COLA_FORCE_LAYOUT:
-      return new TopologyColaLayout(graph, { layoutOnDrag: true });
     case COLA_LAYOUT:
```
Comment (debsmita1) [inline frontend/packages/topology/src/components/graph-view/layouts/layoutFactory.ts:8]: do we need the switch case anymore ?

Comment (abhinandan13jan) [reply frontend/packages/topology/src/components/graph-view/layouts/layoutFactory.ts]: removed

**Review by vikram-raj — COMMENTED** [review]
/lgtm

**Review by jerolimov — APPROVED** [review]
Tested it locally and works fine.

/approve

**Comment by invincibleJai** [comment]
/label docs-approved
/label px-approved

propagating above labels based on ACKs on epic

cc @rishumehra

**Pull request merged**